tests/test_opds.py

```diff
@@ -327,206 +313,251 @@ def test_adobe_id_tags_when_vendor_id_configured(self):
             assert same_tag is not element
             eq_(etree.tostring(element), etree.tostring(same_tag))
 
-            
+
 class TestOPDS(WithVendorIDTest):
 
     def setup(self):
         super(TestOPDS, self).setup()
         parent = Lane(self._db, "Fiction", languages=["eng"], fiction=True)
-        fantasy_lane = Lane(self._db, "Fantasy", languages=["eng"], genres=[Fantasy], parent=parent)
-        self.lane = fantasy_lane
+        self.lane = Lane(self._db, "Fantasy", languages=["eng"], genres=[Fantasy], parent=parent)
+        self.annotator = CirculationManagerAnnotator(None, self.lane, test_mode=True)
```

Comment:
I refactored many of the tests below to use this annotator instead of creating one.